arbos/parse_l2.go

```diff
@@ -182,6 +182,9 @@ func parseL2Message(rd io.Reader, poster common.Address, timestamp uint64, reque
 		return nil, errors.New("L2 message kind SignedCompressedTx is unimplemented")
 	default:
 		// ignore invalid message kind
+		if len(l2KindBuf) < 1 {
```

Comment:
it's an extreme nit, but, at least for my brain, parsing `if len(x) == 0` is a bit faster and requires less thinking than `if len(x) < 1`